.circleci/config.yml

```diff
@@ -0,0 +1,151 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/ui-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
```

Comment:
is this something we will have to manually bump in the future?

Reply:
Generally not. However it has prooven to be helpful to not only rely on the checksum or the yarn.lock. Whenever CircleCI's cache needs to be pruned we can just bump that v{x} thing.